src/Arcus.Scripting.ApiManagement/Arcus.Scripting.ApiManagement.psd1

```diff
@@ -48,7 +48,7 @@ Description = 'Scripts related to Azure API Management'
 # ProcessorArchitecture = ''
 
 # Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @('Az.ApiManagement')
+RequiredModules = @("Az.ApiManagement")

```

Comment:
Apperently, adding these required modules to the manifest throws an exception I can't yet figure out how to solve. Any ideas?

Reply:
Maybe it's the format? Azure PowerShell does it like this, but not sure if it's custom or not - https://github.com/Azure/azure-powershell/blob/master/src/ApiManagement/ApiManagement/Az.ApiManagement.psd1#L56

Reply:
Yeah, I tried different formats. Normally just a string just be correct too. I changed it as in the link you send... same result 😭 . Not sure what I'm doing wrong here.

Reply:
What I would do to improve our modules is:
- Include `PowerShellVersion` (see [sample](https://github.com/Azure/azure-powershell/blob/master/src/ApiManagement/ApiManagement/Az.ApiManagement.psd1#L38))
- Add `Tags`, `LicenseUri`, `ProjectUri`, `IconUri` in private data (see [sample](https://github.com/Azure/azure-powershell/blob/master/src/ApiManagement/ApiManagement/Az.ApiManagement.psd1#L186))

Reply:
https://github.com/arcus-azure/arcus.scripting/issues/65